src/js/models/LottosModel.js

```diff
@@ -0,0 +1,17 @@
+import Lotto from './Lotto';
+import { checkValidMoneyInput } from '../utils/Lotto/validator';
+import { LOTTO_SETTING } from '../constants/setting';
+
+export default class LottosModel {
+  #lottos = [];
+
+  buy(inputMoney) {
+    checkValidMoneyInput(inputMoney);
+    const lottoCount = inputMoney / LOTTO_SETTING.PRICE;
+    Array.from({ length: lottoCount }, () => this.#lottos.push(new Lotto().generate()));
```

Comment:
Array.from 을 이용하신 거 멋지네요 👍 

근데 개인적으로 generate 함수가 로또번호를 생성하고, this 까지 반환하는 게 좀 어색하다고 느껴지는데 this 까지 반환하는 이유는 여기서 바로 넣기 위함일까요? 어떤 의도가 있을까용?
```suggestion
    // 제가 만약 작성한다면 아래와 같이 해볼까?싶습니다!
    Array.from({ length: lottoCount }, () => {
      const LottoGame = new Lotto()
      LottoGame.generateNumbers()
      this.#lottos.push(LottoGame)
    });
```